modules/node_modules/@frogpond/ccc-wpjson-feed/index.js

```diff
@@ -8,6 +8,13 @@ export async function fetchWpJson(url, query = {}) {
 }
 
 export function convertWpJsonItemToStory(item) {
+	let categories
+	if (item._embedded && item._embedded['wp:term']) {
```

Comment:
They'll never be null, since this is called as an array.map callback on the data from the feed. 

The PR is relying on the output from Wordpress, which is quite stable. 

I don't think we need to worry about null items or non-object _embed keys.

Reply:
Are you thinking of an overall check before doing anything with `item`? So before line 11, something like

```js
if (!item) {
  return {}
}
```

or are you saying to check each time we are trying to index into `item`?

```js
if (item && item._embedded && item._embedded['wp:term']) {
```

I'm not sure that we actually care to check if `item` is null, though. What do you think, @hawkrives?

Reply:
My primary proposal is to put it into the `if`, per your second suggestion.

```js
if (item && item._embedded && item._embedded['wp:term']) //...
```

But this feedback is actually out-of-scope, because down below we're calling `item.author` which already has the issue I mentioned. Since I'm doubtful I'll convince myself and everyone else that every function should be able to be run in isolation (i.e. in tests, with arbitrary input), this is fine. Carry on.

Reply:
I do appreciate defensive programming. Feel free to raise an issue about this.